app/models/group.js

```diff
@@ -15,6 +15,11 @@ exports.addModel = function(database) {
     this.screenName = params.screenName
     this.createdAt = params.createdAt
     this.updatedAt = params.updatedAt
+    if (params.hasOwnProperty('visibility')) {
```

Comment:
I would rather keep the visibility, invite and posting rights separate. Visibility also applies to users, and I would like to reuse the same logic for checking timeline/post visibility between users and groups.